src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,106 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int INVALID_AGE = 16;
+    private static final int VALID_AGE = 18;
+    private static final String INVALID_PASSWORD = "qwer";
+    private static final String VALID_PASSWORD = "qwerty";
+    private static final String VALID_LOGIN = "nameEx";
+    private static final String INVALID_LOGIN = "name";
+
+    private User user;
+    private StorageDao storage = new StorageDaoImpl();
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setAge(VALID_AGE);
+        user.setPassword(VALID_PASSWORD);
+        user.setLogin(VALID_LOGIN);
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
```

Comment:
This should be done in BeforeEach to make sure that db is clear before test execution. For example other tests can use storage as well, and if they don`t clean up, test that you wrote could be broken